### PR TITLE
docs: update

### PR DESCRIPTION
**Pull request by rr-**
#### Checklist

- [X] I have read the [coding conventions](https://github.com/rr-/Tomb1Main/blob/master/CONTRIBUTING.md#coding-conventions)
- [X] I have added a changelog entry about what my pull request accomplishes, or it is an internal change

#### Description

- Added a logo
- Moved showcase earlier  
  Can someone prepare new screenshots and texts? I'd say 6 screenshots would be ideal.
- Described how to install the game using the new installer (it's not yet released!)
- Simplified the advanced installation instructions
- Split the improvements section into areas of interest
- Simplified the roadmap, made it less whiny

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
+<p align="center">
+<img alt="Tomb1Main logo" src="tools/resources/logo.png" width="400"/>
+</p>
+
 # Tomb Raider I: Community Edition
 
-This is an open source implementation of the classic Tomb Raider I game. It was
+This is an open source implementation of the classic Tomb Raider I game,
```
Comment (Richard-L) [inline README.md]: maybe add "(1996)" for clarity

File: README.md [hunk 2]
```diff
@@ -11,47 +15,66 @@ This project was inspired by Arsunt's
...
+            <img src="docs/showcase5.jpg"/>
+        </th>
+    </tr>
+</table>
+
+## Installing (simplified)
+
+1. Head over to GitHub releases: https://github.com/rr-/Tomb1Main/releases
+2. Download the installer. Your browser may complain that the .exe is unsafe, but it's OK to ignore this alert.
+3. Mark the installer EXE as safe to run by right-clicking on the .exe, going to properties and clicking "Unblock".
```
Comment (Richard-L) [inline README.md:63]: Don't need this line imo. It won't say "Unblock" for Win7 users for example, it's called something else.

File: README.md [hunk 3]
```diff
@@ -11,47 +15,66 @@ This project was inspired by Arsunt's
...
-    1. Create a shortcut to `Tomb1Main.exe`.
-
-    2. Select the newly created shortcut, go to File Properties.
-
-    3. At the end of the "target" field, append `-gold` so it looks something like
-        this:  
-        `(...)\Tomb1Main.exe -gold`
-
-    4. Launch the shortcut.
+   (`cfg/Tomb1Main_config.json5` can remain).
```
Comment (Richard-L) [inline README.md]: perhaps add "..but newly added features won't be configurable".

File: README.md [hunk 4]
```diff
@@ -11,47 +15,66 @@ This project was inspired by Arsunt's
...
+4. (First time installation) Put your original game files into the target directory.
+    1. For Steam and GOG users, extract the original `GAME.BIN` file using a tool such as UltraISO to your target directory.
+       Note that neither the GOG nor the Steam releases ship the music files. You have a few options here:
+       - You can download the music files from the link below.  
+         https://tmp.sakuya.pl/tomb1main/music.zip
+         The legality of this approach is disputable.
+       - Rip the assets yourself from a physical PlayStation/SegaSaturn disk.
+       Optionally you can also install the Unfinished Business expansion pack files.
```
Comment (Richard-L) [inline README.md:83]: This line should receive its own bullet point, or be somehow linked with the two UB points following. It's currently part of the instruction on audio ripping.

File: README.md (hunk 2 above)
Comment (rr-) [reply README.md:63]: Microsoft's support for Windows 7 ended on January 14, 2020

File: README.md (hunk 3 above)
Comment (rr-) [reply README.md]: Added

File: README.md (hunk 1 above)
Comment (rr-) [reply README.md]: Added

File: README.md (hunk 4 above)
Comment (rr-) [reply README.md:83]: Fixed

**Comment by walkawayy** [comment]
Will Markdown play gifs? That would be a good opportunity to show off some features that need motion.

Here's a list of some features I think would be cool to show off:

1. PS1 style UI
2. Fade effects
3. Restart / select level
4. Compass / final stats
5. Enemy health bars
6. Fly / turbo cheats
7. Before and after pics to show fixed water caustics and/or `fixed thin black lines between polygons` (aka the black lines all over the walls)
8. Walk to items
9. Lost Valley braid (would be nicer to showcase if it worked in all levels)

Those features jump out to me as good image / gif showcases. Stuff like controller support is a key feature but is there a good way to show it off here?

Approved so that it can be merged once the showcase is finalized. I like the new readme format.

**Review by walkawayy — APPROVED** [review]

**Comment by Richard-L** [comment]
Are GIF still better because people don't need to click? They can quickly get really big in size, and MP4 can be embedded on GH.

I had begun drafting a feature table overview for TR1M once, grouped into columns of "Features from": PS1, Saturn, DOS, TombATI. Then "Fixes from": TombATI, etc.
If there's interest I could finish that.

To showcase controller support we could use the controller overlays streamers use, for example here https://youtu.be/NhfjGrahdU4. There's also one for Xbox controllers, and for keyboards.

**Comment by rr-** [comment]
I'd appreciate if someone donated webm/mp4/webp/gif showcase. They can be animated.
Max 8 images, selected in terms of importance on the gameplay. Having a table with all the features pictured would be too overwhelming.

**Comment by walkawayy** [comment]
Maybe @Richard-L and I can coordinate something in the Discord channel. Anything specific you want to show off?

**Comment by rr-** [comment]
I'd say the fly cheat and enemy health bars, maybe improved stats, maybe load from any level.

**Comment by rr-** [comment]
Compressed images in size from 7 MB to 1 MB.
Zoomed in the 3D Pickups screenshot to show the pickup better.
Renamed the files.
Retaken the fog distance picture in Collosseum.

**Comment by rr-** [comment]
@walkawayy bump for a review

**Comment by rr-** [comment]
@walkawayy bump for a review

**Comment by walkawayy** [comment]
Sorry about that! I meant to post over the weekend but I've been dealing with some irl stuff. The only two things I noticed were the images are two per row except the 8th image (select level) is missing. So the last row only has 1 of the 2 images.

Also for the ps1 UI image, you could rename it to say something like "ps1 UI and new graphics options" or something. I used that image to show off two features (shows the scaling, vsync, high res, etc). The title of that screenshot was a WIP name.

Otherwise I thought the images look great! Love the animated draw distance one.

**Comment by rr-** [comment]
Updated wording, IMO it's OK to keep one slot free in case we land something amazing.

**Comment by walkawayy** [comment]
Np. Just wanted to point in out in case it was a mistake. If you want to showcase a different feature go ahead imo. You could always remove it if a new feature develops down the road though.

(Btw I already approved so merge whenever you are ready 😃 . Looks much nicer than the old format which had grown a lot.)

**Pull request merged**